tests/foreman/api/test_contentmanagement.py

```diff
@@ -1463,3 +1464,36 @@ def test_positive_sync_file_repo(
             sat_file = md5_by_url(f'{sat_repo_url}{file}')
             caps_file = md5_by_url(f'{caps_repo_url}{file}')
             assert sat_file == caps_file
+
+    @pytest.mark.tier3
+    def test_positive_allow_reregistration_when_dmi_uuid_changed(
+        self, module_org, rhel_contenthost, default_sat
+    ):
+        """Register a content host with a custom DMI UUID, unregistering it, change
+        the DMI UUID, and re-registering it again
+
+        :id: 7f431cb2-5a63-41f7-a27f-62b86328b50d
+
+        :expectedresults: The content host registers successfully
+
+        :customerscenario: true
+
+        :BZ: 1747177
+
+        :CaseLevel: Integration
+        """
+        uuid_1 = str(uuid.uuid1())
+        uuid_2 = str(uuid.uuid4())
+        rhel_contenthost.install_katello_ca(default_sat)
+        default_sat.execute(
+            f'echo \'{{"dmi.system.uuid": "{uuid_1}"}}\' > /etc/rhsm/facts/uuid.facts'
```

Comment:
This f-string will technically call `str` on the variable that's being formatted, so a bit redundant wrapping it ahead of time, but doesn't hurt?